tools/docker/.env

```diff
@@ -13,6 +13,7 @@ MINIMUM_CONTRACT_PAYMENT=1000000000000
 MIN_INCOMING_CONFIRMATIONS=1
 MIN_OUTGOING_CONFIRMATIONS=1
 RUST_BACKTRACE=1
+DEFAULT_HTTP_TIMEOUT=5s
```

Comment:
Why overwrite the default?

Reply:
This is necessary to make the tests run faster because the `pollingInterval` for FM jobs has to be >= the timeout and we want the polling Interval to be small for fast testing (it's currently set to 5s in the integration tests)